src/components/Account/Create.tsx

```diff
@@ -50,32 +48,22 @@ export const AccountCreate = ({ children, ...props }: FlexProps) => {
       description: '',
     },
   })
+
   const {
-    createAccount,
-    updateAccount,
     errors: { account: error },
   } = useClient()
-  const { t } = useTranslation()
+
+  const { mutateAsync } = useAccountCreate()
```

Comment:
@selankon since the `createAccount` funcionality is now being done by the `useAccountCreate` hook shouldn't it expose as well the errors instead of importing them from `useClient`?

Reply:
Mmm actually not, the errors came from the client because the functions called inside the `useAccountCreate` hook, `createAccount, updateAccount`, are processed on the react providers, and the errors are handled there. 

So, if the `createAccount, updateAccount` no error is thrown, and the errors are exposed on the `useClient` provider, afaik. 

So the mutation should never fail, the errors are handled inside the provider and updated the state with it.

Now talking about this, maybe the `useAccountCreate`  should return just the `create` function as a `useCallback` instead of wrap it in a mutation?

Reply:
We decided to don't use a mutation, use a hook instead that expose the errors of the useClient

Reply:
useAccountCreate